<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculatrice</class>
 <widget class="QMainWindow" name="Calculatrice">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>260</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Calculatrice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="addTab">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCalvier" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>531</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>511</width>
       <height>125</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <item row="0" column="4">
       <layout class="QGridLayout" name="numLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="1" column="1">
         <widget class="QPushButton" name="Bt8">
          <property name="text">
           <string>8</string>
          </property>
          <property name="shortcut">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="Bt9">
          <property name="text">
           <string>9</string>
          </property>
          <property name="shortcut">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="Bt4">
          <property name="text">
           <string>4</string>
          </property>
          <property name="shortcut">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Bt5">
          <property name="text">
           <string>5</string>
          </property>
          <property name="shortcut">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Bt6">
          <property name="text">
           <string>6</string>
          </property>
          <property name="shortcut">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="Bt1">
          <property name="text">
           <string>1</string>
          </property>
          <property name="shortcut">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="Bt2">
          <property name="text">
           <string>2</string>
          </property>
          <property name="shortcut">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="Bt3">
          <property name="text">
           <string>3</string>
          </property>
          <property name="shortcut">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="Bt0">
          <property name="text">
           <string>0</string>
          </property>
          <property name="shortcut">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="BtMult">
          <property name="text">
           <string>*</string>
          </property>
          <property name="shortcut">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="BtDiv">
          <property name="text">
           <string>/</string>
          </property>
          <property name="shortcut">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="BtSigne">
          <property name="text">
           <string>(-)</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="BtVirgule">
          <property name="text">
           <string>,</string>
          </property>
          <property name="shortcut">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="BtExpression">
          <property name="text">
           <string>'</string>
          </property>
          <property name="shortcut">
           <string>'</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Bt7">
          <property name="text">
           <string>7</string>
          </property>
          <property name="shortcut">
           <string>7</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="BtMoins">
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="BtPuissance">
          <property name="text">
           <string>^</string>
          </property>
          <property name="shortcut">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Pavé numérique:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="BtEspace">
          <property name="text">
           <string>_</string>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="BtPlus">
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="BtEval">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Eval</string>
          </property>
          <property name="shortcut">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="pileLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="2" column="0">
         <widget class="QPushButton" name="BtSWAP">
          <property name="text">
           <string>SWAP</string>
          </property>
          <property name="shortcut">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="BtSUM">
          <property name="text">
           <string>SUM</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="BtMEAN">
          <property name="text">
           <string>MEAN</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="BtCLEAR">
          <property name="text">
           <string>CLEAR</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="BtDUP">
          <property name="text">
           <string>DUP</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="BtDROP">
          <property name="text">
           <string>DROP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Opération sur la pile:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="fonctionLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="2" column="0">
         <widget class="QPushButton" name="BtSIN">
          <property name="text">
           <string>SIN</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="BtCOS">
          <property name="text">
           <string>COS</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="BtTAN">
          <property name="text">
           <string>TAN</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="BtSINH">
          <property name="text">
           <string>SINH</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="BtCOSH">
          <property name="text">
           <string>COSH</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="BtTANH">
          <property name="text">
           <string>TANH</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="BtLN">
          <property name="text">
           <string>LN</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="BtLOG">
          <property name="text">
           <string>LOG</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="BtSQRT">
          <property name="text">
           <string>SQRT</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="BtSQR">
          <property name="text">
           <string>SQR</string>
          </property>
          <property name="shortcut">
           <string>²</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="BtCUBE">
          <property name="text">
           <string>CUBE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="BtFactoriel">
          <property name="text">
           <string>!</string>
          </property>
          <property name="shortcut">
           <string>!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="BtComplexe">
          <property name="text">
           <string>$</string>
          </property>
          <property name="shortcut">
           <string>$</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Fonction:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="BtModulo">
          <property name="text">
           <string>%</string>
          </property>
          <property name="shortcut">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrée:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="angleTypeLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_Degre">
       <property name="text">
        <string>Degre</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Radian">
       <property name="text">
        <string>Radian</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>299</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="entreeTypeLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_Entier">
       <property name="text">
        <string>Entier</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Rationnel">
       <property name="text">
        <string>Rationnel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Reel">
       <property name="text">
        <string>Reel</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="nbElementPile">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="BtEnvoyer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entreeTxt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtVider">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vider</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtBack">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="complexeBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Complexe</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKjg">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRetablir"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionClavier"/>
   </widget>
   <addaction name="menuKjg"/>
   <addaction name="menuAffichage"/>
  </widget>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRetablir">
   <property name="text">
    <string>Rétablir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionClavier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clavier</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionClavier</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetCalvier</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
